src/pages/about.tsx

```diff
@@ -1,127 +1,118 @@
 import { Link } from 'next-theme-ui';
```

Comment:
Seems like this file had some unicode issues.

https://github.blog/changelog/2021-10-31-warning-about-bidirectional-unicode-text/

Reply:
Hmm, there was a weird unicode character in one of the ContactPerson entries that I removed - but I am unsure how to fix this.

Reply:
Let's just ignore it :)